### ISSUE TITLE
Accessibility violations: theme modifies outline styles for focused elements

### ISSUE THREAD

**Issue by jessepearson** [woocommerce/storefront#918]
The below is copied from this forum thread:
https://wordpress.org/support/topic/accessibility-violations-theme-modifies-outline-styles-for-focused-elements/

=========

This theme modifies the CSS outline styles for focus elements and makes the focus extremely challenging, if not impossible, to see.

When those who cannot use a mouse navigate these sites via keyboard, or other assistive device, they are dependent on these styles to detect which items have focus. These tiny dashed lines, or simply changing the contrast of inputs, are barely visible to users with good vision, much less those with low vision or color blindness.

The base CSS needs to be modified to set more prominent focus styles.

You’ve also removed the text-decoration from all anchors, which causes another accessibility violation. Information cannot be conveyed solely by color. With your current styles, users who are color blind to blue, cannot see that text has a link. All they see is text that looks just like everything else.

The base CSS needs to be modified to not remove text-decoration for all elements.

**Comment by tiagonoronha** [comment]
https://github.com/woocommerce/storefront/pull/1076

**Comment by bamadesigner** [comment]
Thank you!

**Comment by xrahat2011** [comment]
> Thank you!

My site is still going through this problem, can someone please have a look?
wish2print.co.uk
